pkgs/by-name/hy/hyprpanel/package.nix

```diff
@@ -0,0 +1,135 @@
+{
+  lib,
+  system,
+  config,
+  ags,
+  astal,
+  bluez,
+  bluez-tools,
+  brightnessctl,
+  btop,
+  dart-sass,
+  fetchFromGitHub,
+  fetchpatch2,
+  glib,
+  gnome-bluetooth,
+  gpu-screen-recorder,
+  gpustat,
+  grimblast,
+  gvfs,
+  hyprpicker,
+  libgtop,
+  libnotify,
+  matugen,
+  networkmanager,
+  python3,
+  pywal,
+  swww,
+  upower,
+  wireplumber,
+  wl-clipboard,
+  writeShellScript,
+
+  enableCuda ? config.cudaSupport,
+}:
+ags.bundle {
+  pname = "hyprpanel";
+  version = "unstable-2025-02-06";
+
+  __structuredAttrs = true;
+  strictDeps = true;
+
+  src = fetchFromGitHub {
+    owner = "Jas-SinghFSU";
+    repo = "HyprPanel";
+    rev = "6b846b97097ba6cae627630ead88774b15af887e";
+    hash = "sha256-1zCjMuRrxrsFgl8FH/pjwyM9dxoiQDc3SgrjGqOGv/c=";
+  };
+
+  patches = [
+    # please merge https://github.com/NixOS/nixpkgs/pull/374302
+    ./remove-tray.patch
```

Comment:
Please merge https://github.com/NixOS/nixpkgs/pull/374302 so I don't have to remove tray functionality